bookshelf/Shelf 1 Getting Started/overview/src/README.md

```diff
@@ -0,0 +1,33 @@
+# Open Device Partnership Overview
+![Open Device Partnership](media/odp.png)
+
+## Overview
+An alliance of industry-leading PC ecosystem partners promoting secure, reusable, and trusted system software for client devices
+
+The Open Device Partnership (ODP) is an open-source initiative focused on:
+- enhancing device security
+- simplifying cross-architecture development (ARM & x86 standards)
+- strengthening fundamentals (“raise all boats”)
+- accelerating the delivery of high-quality devices
+
+Designed from the onset to be inclusive of all device ecosystem partners and stakeholders
+- one stop location for everything needed to build a Windows client device
+
+Optimized for Windows devices but only successful if Linux can also thrive
+
+Open-source (MIT) with open governance focused on driving ecosystem innovation
+
+Long time-horizon investment (start by meeting partners where they are today)
+
+## Projects
```

Comment:
Minor point but it would be nice to mention that these are just the current project and that we will be adding new projects (via the ODP governance process) that are aligned with our shared objectives...